site/assets/scss/_content.scss

```diff
@@ -31,15 +31,12 @@
   }
 
   // Override Bootstrap defaults
-  > .table {
-    max-width: 100%;
+  > .table,
+  > .table-responsive .table {
     margin-bottom: 1.5rem;
     @include font-size(.875rem);
```

Comment:
@ffoodd @mdo: perhaps we could increase this a bit? I feel it's too small right now.

Reply:
Goes from 16px default font size down to only 14px. I think it works really well for the type of content here.